### PR TITLE
feat(suite) account menu UX plus ERC20

### LINKED ISSUE #3747: Better UX/UI: Account menu + ERC20
**Design:** [Zeplin](https://zpl.io/Vk148OZ) 

**This release (June 9th):** 

- [x] Tag on ETH indicating number of individual Tokens 
- [x] Tab `Tokens` next to the Overview tab on ETH _(Zeplin to be added here)_
- [x] Add account button
- [x] Fonts (weight, size etc.)   
- [x] Hover + Selected Account state: [Zeplin](https://zpl.io/ad3398E)

**Future release (July 14th):** 

- Un/Archive + Un/Favourite + Sort #3748

### PR DESCRIPTION
**Pull request by matejkriz**
close #3747

Please look at `This release (June 9th)` part of the related issue for the description.

### CODE REVIEW

**Comment by tomexx** [comment]
Let's center it vertically so we won't trigger people with OCD :)
image

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    });
+    const theme = useTheme();
+    return (
+        <ButtonBadge
+            icon="PLUS"
+            variant="tertiary"
+            size={10}
+            color={theme.TYPE_LIGHT_GREY}
+            onClick={() => {
+                goto('wallet-tokens', undefined, true);
```
Comment (tomexx) [inline packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx]: I personally think the token badge should not be this clickable (you might discuss this with @matejzak), but I would not expect it there.

I recommend:
- disable hover color change for the token badge
- remove shortcut to tokens

If you and @matejzak decide to keep it clickable, please take a look at the buggy behaviour which is in place with current solution.  Steps to reproduce below.

Buggy behaviour - steps to reproduce:
- select account without tokens (e.g. BTC)
- then click on token badge in ETH account
- you will end up on "No tokens... yet." screen which is invalid for BTC and other accounts w/o tokens support

**Review by tomexx — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx (hunk 1 above)
Comment (matejkriz) [reply packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx]: Discussed with Ondra and @matejzak, they like the badge to be clickable, but I'm not sure about the hover color. And definitely I will fix the bug, thanks for the catch!

Comment (matejkriz) [reply packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx]: @tomexx bug fixed, could you please check again?

**Review by tomexx — APPROVED** [review]
looks good to me now :)

**Comment by tomexx** [comment]
just squash fixups please .)

**Comment by matejkriz** [comment]
There is one more design request from @matejzak :

- [x] remove header "Tokens" inside Tokens tab (tab title is enough)

**Pull request merged**